formspree/templates/email/pre_inline_style/overlimit-notification.html

```diff
@@ -38,7 +38,7 @@
 									</tr>
 									<tr>
 										<td class="content-block">
-											You are past our limit of {{ config.MONTHLY_SUBMISSIONS_LIMIT }} submissions per month. Never fear! We have safely stored the contents of this form in our databases, and you can access them by logging into your account, using the link below and upgrading your account.
+											You are past our limit of {{ limit }} submissions per month. Never fear! We have safely stored the contents of this form in our databases, and you can access them by logging into your account, using the link below and upgrading your account.
```

Comment:
You are past the limit of {{ limit }} free submissions per month. Never fear! We have safely stored the content of this form in our archive. You can access it by logging in at {{ url_for('register') }} and upgrading your account.